src/mmw/apps/modeling/tasks.py

```diff
@@ -108,17 +108,18 @@ def get_histogram_job_results(self, incoming):
 
 
 @shared_task
-def histogram_to_survey(incoming, aoi):
+def histogram_to_survey_census(incoming):
```

Comment:
You need to keep two parameters here. The first parameter is automatically supplied by the previous task in the Celery chain. The second argument is the area of interest specified in views.py:306 below.